### PR TITLE
Spanish Language Support

### PR DESCRIPTION
**Pull request by beausmith**
@eventualbuddha - @benadida and I discussed using this format for language strings such that we don't break apps using the existing election definition format. Open to all comments/thoughts/etc.

### CODE REVIEW

File: src/data/electionSample.json [hunk 1]
```diff
@@ -3,9 +3,20 @@
...
+      "es-us": {
+        "name": "Franklin County"
+      }
+    }
+  },
+  "date": "2020-11-03",
+  "_lang": {
+    "es-us": {
+      "title": "Eleccion General",
+      "state": "State of Hamilton"
```
Comment (benadida) [inline src/data/electionSample.json]: is this the right string?

**Review by benadida — APPROVED** [review]

File: src/data/electionSample.json (hunk 1 above)
Comment (eventualbuddha) [inline src/data/electionSample.json]:
```suggestion
      "state": "Estado de Hamilton"
```

**Review by eventualbuddha — APPROVED** [review]
One correction, which @benadida already pointed out.

File: src/data/electionSample.json (hunk 1 above)
Comment (beausmith) [reply src/data/electionSample.json]: Thanks. Caught it as I was resting and not getting expected result 🙃

**Pull request merged**